ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[hclecomm.logo,hclecomm.structure]"/>
```

Comment:
@DeepaliAgrwl  Keep only one category -hclecomm.logo and update other files accordingly

Reply:
@rakeshkumar-verma removed one category. keeping one category as heclecomm.logo

Reply:
also changed the respective base-clientlib for this